templates/SilverStripe/ElementalBannerBlock/Block/BannerBlock.ss

```diff
@@ -1,27 +1,32 @@
-<%-- Display the image (File) --%>
-<% if $File %>
-    <span class="banner-element__image">
-        $File
-    </span>
-
-<% end_if %>
-<div class="banner-element__content">
-    <% if $Title && $ShowTitle %>
-        <h2 class="banner-element__title">$Title</h2>
-    <% end_if %>
+<a class="banner-block"
```

Comment:
Linking the whole block breaks text selection. That's not ideal, but Paul specifically asked for the whole block to be linked, so fine with me.